src/libs/actions/BankAccounts.js

```diff
@@ -270,12 +274,359 @@ function fetchUserWallet() {
         });
 }
 
+let previousACHData = {};
+Onyx.connect({
+    key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+    callback: (val) => {
+        previousACHData = lodashGet(val, 'achData', {});
+    },
+});
+
+/**
+ * Navigate to a specific step in the VBA flow
+ *
+ * @param {String} stepID
+ * @param {Object} achData
+ */
+function goToWithdrawalAccountSetupStep(stepID, achData) {
+    const newACHData = {...previousACHData};
+
+    // If we go back to Requestor Step, reset any validation and previously answered questions from expectID.
```

Comment:
NAB: `When going to the Requestor Step, reset any validation and previously answered questions from expectID.`